<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_widget</class>
 <widget class="QWidget" name="add_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>                              Add a Pioneer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_firstname">
         <property name="placeholderText">
          <string>Firstname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_lastname">
         <property name="placeholderText">
          <string>Lastname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_gender">
         <property name="placeholderText">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input__yearOfBirth">
         <property name="placeholderText">
          <string>Year of birth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_yearOfDeath">
         <property name="placeholderText">
          <string>Year of death</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_pioneer">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Add a Computer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_computer_name">
         <property name="placeholderText">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_type">
         <property name="placeholderText">
          <string>type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_yearBuilt">
         <property name="placeholderText">
          <string>Built(0 if not built</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_ifMade">
         <property name="placeholderText">
          <string>made(0 if made, else 1)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_computer">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Add a Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_connect_lastname">
         <property name="placeholderText">
          <string>Lastname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_connect_firstname">
         <property name="placeholderText">
          <string>Firstname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Input_connect_computer_name">
         <property name="placeholderText">
          <string>Name of computer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_connection">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
